geometry_msgs/Point point
bool isManipulable
uint32 category
